codespell_lib/data/dictionary.txt

```diff
@@ -1371,6 +1371,7 @@ allwys->always
 almostly->almost
 almsot->almost
 alo->also
+aloable->allowable
```

Comment:
This seems quite a stretch!

The only hit grep.app had was avaloable which should be to available so would you mind adding that?

Reply:
stretch, you say? github says otherwise: https://github.com/search?q=%22aloable%22&type=code

Reply:
Okay, although they seem to be one typo in lots of forks of the same repo. I guess it's fixed upstream hence why grep.app isn't returning it or something?

Anyway fair enough.